tests/playwright-test/golden.spec.ts

```diff
@@ -696,12 +696,16 @@ test('should respect comparator in config', async ({ runInlineTest }) => {
   expect(result.report.suites[0].specs[0].tests[1].status).toBe('unexpected');
 });
 
+// ../../snapshot!.txt as a snapshot name.
+// Its different per platform because we normalize the slashes.
```

Comment:
This is problematic - we want the same file names on different OSes, because they are committed to a repo.

Reply:
clarified offline, don't touching vrt snapshot filenames.